zhaquirks/ikea/__init__.py

```diff
@@ -209,26 +209,36 @@ async def bind(self):
         await self.endpoint.basic.read_attributes([Basic.AttributeDefs.sw_build_id.id])
         return result
 
-    def _is_firmware_old(self):
-        """Checks if firmware is old or unknown."""
+    def _is_firmware_new(self):
+        """Checks if new firmware is installed that does not require battery doubling."""
         # get sw_build_id from attribute cache if available
         sw_build_id = self.endpoint.basic.get(Basic.AttributeDefs.sw_build_id.id, None)
```

Comment:
```suggestion
        sw_build_id = self.endpoint.basic.get(Basic.AttributeDefs.sw_build_id.id)
```
For a future PR: `None` is already the default for zigpy clusters, so we can remove that.